src/fides/api/models/application_config.py

```diff
@@ -169,6 +169,20 @@ def clear_api_set(cls, db: Session) -> Optional[ApplicationConfig]:
             return existing_record
         return None
 
+    @classmethod
+    def clear_config_set(cls, db: Session) -> Optional[ApplicationConfig]:
```

Comment:
not strictly needed, but figured i'd add because it was helpful for a test and gives us a more complete API on this model!